libs/partners/ai21/tests/integration_tests/test_compile.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+
+@pytest.mark.compile
+def test_placeholder() -> None:
```

Comment:
Can this be removed?

Reply:
I don't think so. They have a github action where they run a integration test "compilation"
![image](https://github.com/AI21Labs/langchain/assets/147075902/28755cda-a796-4d96-a9cb-d363a6563327)